src/js/sdk.ts

```diff
@@ -24,6 +25,7 @@ const DEFAULT_OPTIONS: ReactNativeOptions = {
   enableNativeCrashHandling: true,
   enableNativeNagger: true,
   autoInitializeNativeSdk: true,
+  enableStallTracking: true,
```

Comment:
One thought, do we want to make it not default enabled for this release, ask users to try it out manually, and then enable by default in the release after that?

Reply:
@bruno-garcia What do you say. Although this would be released in a beta, if we just enable it, it might work because the people who would be trying this out would be doing so just for it anyways? I'm fine with either.